stdlib/email/src/main/java/org/ballerinalang/stdlib/email/service/compiler/EmailListenerCompilerPlugin.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.stdlib.email.service.compiler;
+
+import org.ballerinalang.compiler.plugins.AbstractCompilerPlugin;
+import org.ballerinalang.util.diagnostic.DiagnosticLog;
+
+/**
+ * This class validates annotations attached to the Ballerina service and its resource nodes.
+ *
+ * @since 1.1.5
```

Comment:
This will be added to master right? The version would have to be fixed everywhere as 1.2.0.

Reply:
This will be released with 1.1.5.

Reply:
Then the PR should be point to 1.1.x branch. Isn't it?

Reply:
Yes. Will send another PR to 1.1.x branch.